code/datums/components/health_scaling_effects.dm

```diff
@@ -0,0 +1,75 @@
+/**
+ * ### Enrage buffs component
+ *
+ * Scales some statistics of a living mob (speed or attack power or such) based on how hurt it is.
+ */
+/datum/component/health_scaling_effects
+	var/max_health_threshold
+	var/min_health_threshold
+	var/max_health_attack_modifier_lower
+	var/min_health_attack_modifier_lower
+	var/max_health_attack_modifier_upper
+	var/min_health_attack_modifier_upper
+	var/max_health_slowdown
+	var/min_health_slowdown
+	var/datum/callback/additional_status_callback
```

Comment:
doc these vars please, if for no other reason than to make sure our documentation stats stay swagging